repositories/cyclonedds.BUILD.bazel

```diff
@@ -329,6 +332,7 @@ _DDSI_SRCS = [
     "ddsi_vnet.c",  # Manually added!
     "ddsi_wraddrset.c",  # Manually added!
     "ddsi_list_genptr.c",
+    "ddsi_shm_transport.c",
```

Comment:
How about we start using rules_foreign_cc for cyclonedds?

Reply:
You mean like using a cmake rule? Could be simpler to maintain. Not sure I will have time to work on that tho.

Reply:
Already in the main branch.